index.html

```diff
@@ -18,7 +18,7 @@ <h1 class="page-header-title animated fadeInDown"><span class="text-primary">Wom
                     <h5 class="sub-title animated fadeIn">The professional network for women who work in robotics - or
                         who
                         aspire to.</h5>
-                    <a href="https://docs.google.com/forms/d/e/1FAIpQLScbQPmdjz9v5PyF50e0m1nWNUKyNhFfw9JQ_SvlUF25guwPNg/viewform?embedded=true"
+                    <a href="https://womeninrobotics.github.io/signup/"
```

Comment:
Prefer to not use the full url so we can move to www.womeninrobotics.org

```suggestion
                    <a href="/signup/"
```

Reply:
alternatively, use `{{ site.url }}/signup/`